tests/foreman/cli/test_errata.py

```diff
@@ -80,113 +74,107 @@
 from robottelo.constants.repos import FAKE_6_YUM_REPO
 from robottelo.constants.repos import FAKE_9_YUM_REPO
 from robottelo.decorators import skip_if_not_set
-from robottelo.test import CLITestCase
 from robottelo.vm import VirtualMachine
 
+
 ERRATUM_MAX_IDS_INFO = 10
 
+CUSTOM_REPO_URL = FAKE_9_YUM_REPO
+CUSTOM_PACKAGE = FAKE_1_CUSTOM_PACKAGE
+CUSTOM_ERRATA_ID = FAKE_2_ERRATA_ID
+CUSTOM_PACKAGE_ERRATA_APPLIED = FAKE_2_CUSTOM_PACKAGE
+
 
+@skip_if_not_set('clients', 'fake_manifest')
+@pytest.mark.skipif((not settings.repos_hosting_url), reason='Missing repos_hosting_url')
 @pytest.mark.run_in_one_thread
-class HostCollectionErrataInstallTestCase(CLITestCase):
+class TestHostCollectionErrataInstallTest:
     """CLI Tests for the errata management feature"""
 
-    CUSTOM_REPO_URL = FAKE_9_YUM_REPO
-    CUSTOM_PACKAGE = FAKE_1_CUSTOM_PACKAGE
-    CUSTOM_ERRATA_ID = FAKE_2_ERRATA_ID
-    CUSTOM_PACKAGE_ERRATA_APPLIED = FAKE_2_CUSTOM_PACKAGE
-    VIRTUAL_MACHINES_COUNT = 2
 
-    @classmethod
-    @skip_if_not_set('clients', 'fake_manifest')
-    @pytest.mark.skipif((not settings.repos_hosting_url), reason='Missing repos_hosting_url')
-    def setUpClass(cls):
-        """Create Org, Lifecycle Environment, Content View, Activation key,
-        Host, Host-Collection
+@pytest.fixture(scope='module')
```

Comment:
Maybe its just the diff, but these fixture definitions appear to not be indented, and they follow the statement defining the test class.

The fixtures can be defined outside of the class, but they should be defined before the test class.  If you want to limit the fixture resolution to within the class, the fixture functions should be indented to be class methods.

Reply:
I have removed the classes and reformatted the file, added new fixtures to break up the larger ones